### DISCUSSION TITLE
Update on the development branch

### DISCUSSION THREAD

**Discussion by kaiyux** [NVIDIA/TensorRT-LLM#2851] [Announcements]
Hi,

The TensorRT-LLM team is pleased to announce that we have pushed an update to the [development](https://github.com/NVIDIA/TensorRT-LLM/tree/main) branch (and the [Triton backend](https://github.com/triton-inference-server/tensorrtllm_backend/tree/main)) this March 4, 2025.

This [update](https://github.com/NVIDIA/TensorRT-LLM/pull/2849) includes:

* Model Support
  - PyTorch workflow
    - Added Llava-Next support.
    - Added BERT support.
  - Added Qwen2-Audio support. Refer to `examples/qwen2audio/README.md`.
* Features
  - Added Language-Adapter support. Refer to `examples/language_adapter/README.md`.
  - Added AutoDeploy MoE Pattern Matcher for the `fused_moe` op.
  - [BREAKING CHANGE] Added dynamic logits processor support, and deprecated static logits processor in LLM API.
  - Added Deepseek eager mode AllReduce fusion support.
* Bug fixes
  - Fixed incorrect batch slot usage in `addCumLogProbs` kernel. Thanks to the contribution from @aotman in #2787.

Thanks,
The TensorRT-LLM Engineering Team
